lib/src/command/outdated.dart

```diff
@@ -536,9 +536,10 @@ Future<void> _outputHuman(
       upgradable == 0 &&
       rows.isNotEmpty) {
     log.message(
-        '\nCurrent dependencies are constrained to versions listed in Resolvable.'
-        '\nA resolution compatible with the Latest versions doesn\'t exist, as they are not'
-        '\nmutually compatible.');
+        '\nCurrent dependencies are constrained to the versions in the \'Resolvable\' column.'
+        '\n'
+        '\nNo mutually compatible resolution is possible with the versions listed in the'
```

Comment:
I think:
 * This is too strong a statement,
 * A resolution must always be "mutually compatible", so "no mutually compatible" is just extra words..